R/constants.R

```diff
@@ -106,9 +106,12 @@ CONV_KG_TO_TG <- 1e-9
 CONV_M3_BM3 <- 1e-09
 CONV_MILLION_M3_KM3 <- 1e-03
 
+
+
 CONV_M2_ACR <- 0.0002471058
```

Comment:
Sorry, I forgot to check constants.  I don't know why there are so many blank lines.  Everytime I update from master something in the "constants.R" looks crazy.